fastlane/metadata/steam/german/description.txt

```diff
@@ -3,11 +3,13 @@
 [h2]Spielmechanik[/h2]
 
 [list]
-[*] Baue komplexe Produktionsketten aus Förderbändern, um deine Geschütztürme mit Munition zu versorgen
-[*] Stelle Materialien her, um deine Fabrik zu vergrößern und sie vor Gegnerwellen zu schützen
-[*] Spiele mit deinen Freunden in plattformübergreifenden Partien oder fordere sie zum Kampf heraus
-[*] Es gibt immer etwas zu tun: Wenn du deine Kühlsysteme nicht im Blick behältst, könnte dein Reaktor explodieren!
-[*] Produziere Einheiten, die bei deiner Basis helfen oder den Gegner angreifen
+[*] Baue komplexe Produktionsketten aus Förderbändern, um deine Geschütztürme mit Munition zu versorgen und eine große Vielfalt an Ressourcen herzustellen
```

Comment:
```suggestion
[*] Baue komplexe Produktionsketten aus Förderbändern, um deine Geschütztürme mit Munition zu versorgen und erweiterte Ressourcen zum Verbessern deiner Basis herzustellen
```
Auf jeden fall nicht einen nebensatz in aufeinanderfolgenden Punkten wiederholen, vielleicht kann der zweite Teil komplett in den nächsten Punkt, passt hier eigentlich nicht (auch in englicher Version)